Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -50,13 +50,12 @@ GameFile::GameFile(const QString& path) : m_path(path)
 
   if (!TryLoadCache())
```

Comment:
Could this be simplified to `!TryLoadCache() || !TryLoadVolume() || !TryLoadElfDol()`? Putting the checks on three different levels of indentation look kind of weird as they're doing the same thing, just loading from different files.